resources/home/dnanexus/generate_fusion_workbook.py

```
@@ -0,0 +1,104 @@
+
+        # Add SF previous runs data
+        write_df_to_sheet(writer, df_sf_previous, **SF_PREVIOUS_RUNS_SHEET_CONFIG)
+
+        # Add FI previous runs data
+        write_df_to_sheet(writer, df_fi_previous, **FI_PREVIOUS_RUNS_SHEET_CONFIG)
+
+        # Add FastQC data
+        write_df_to_sheet(writer, df_fastqc, **FASTQC_SHEET_CONFIG)
+
+        # Add STAR-Fusion data
+        write_df_to_sheet(writer, df_starfusion, **STAR_FUSION_SHEET_CONFIG)
```

Ruff (0.8.2)

Undefined name `STAR_FUSION_SHEET_CONFIG`

(F821)